test/WebSites/MvcTagHelpersWebSite/Models/Person.cs

```diff
@@ -2,11 +2,13 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNet.Mvc;
 
 namespace MvcTagHelpersWebSite.Models
 {
     public class Person
     {
+        [HiddenInput(DisplayValue = false)]
```

Comment:
note updates here result in the same generated HTML.  in this case, the new `@Html.EditorFor(m => m.Employee.Number)` still generates a hidden input.